### ISSUE TITLE
[QUESTION]: Does the dynamic resume feature work only with OpenAI API?

### ISSUE THREAD

**Issue by himankpathak** [feder-cr/Auto_Jobs_Applier_AIHawk#649]
### Summary of your question

When running the bot, it seems to be crashing when trying to generate resume with gemini API, it looks like lib_resume_builder_AIHawk only accepts OpenAI API key

### Question details

Hi, the bot is running correctly until step
`ai_hawk.linkedIn_easy_applier:_create_and_upload_resume:463 - Generating resume for job:`
where it fails due to the following error:
```
openai.AuthenticationError: Error code: 401 - {'error': {'message': 'Incorrect API key provided: "XYZ". You can find your API key at https://platform.openai.com/account/api-keys.', 'type': 'invalid_request_error', 'param': None, 'code': 'invalid_api_key'}}
```
I looked into it and it seems like the "lib_resume_builder_AIHawk" library only works with OpenAI. If this is true then this needs to be clarified in the Readme to not mislead people to believe that dynamic resume functionality will work with all LLMs.
Please let me know if this is true or did I missed something during setup.

### Context for the question

Occurs at step `ai_hawk.linkedIn_easy_applier:_create_and_upload_resume:463 - Generating resume for job:`

### Additional context

Application called with cmd: `python main.py`

**Comment by FrancescoVassalli** [comment]
As a test can you see if using a resume that you made works? This looks like you are using the API key for the wrong service

**Comment by himankpathak** [comment]
@FrancescoVassalli I tested it and when passing resume with `--resume` arg, it works and successfully applies.

**Comment by 49Simon** [comment]
Yeah this is an issue with the https://github.com/feder-cr/lib_resume_builder_AIHawk library used. It only supports OpenAI when generating custom resume. For now, the workaround is to use your own resume by passing the `--resume` flag.
